config/label.go

```diff
@@ -1,19 +1,23 @@
 package config
 
 import (
+	"fmt"
 	"strconv"
 	"strings"
 
 	microbadger "github.com/microscaling/microbadger/api"
 
 	"github.com/microscaling/microscaling/api"
 	"github.com/microscaling/microscaling/demand"
+	"github.com/microscaling/microscaling/utils"
 )
 
 // LabelConfig is used when we retrieve get image config from the Microscaling server and then
 // get label config from MicroBadger APIs
 type LabelConfig struct {
```

Comment:
It would be more go-ish to have a separate type for the case when there are Kubernetes-specific fields. You already have a separate NewKubeLabelConfig, it would be more typical to have a KubeLabelConfig which supports the same interface as LabelConfig (which doesn't need the k8s-specific fields).